rust/tests/loadtest/src/echo_payload.rs

```diff
@@ -0,0 +1,231 @@
+//! Timestamped echo payload for load testing.
+
+use std::time::{Duration, SystemTime, UNIX_EPOCH};
+
+/// Minimum payload size (connection_id + timestamp).
+pub const HEADER_SIZE: usize = 16;
+
+/// Echo payload format:
+/// - 8 bytes: connection_id (u64, big-endian)
+/// - 8 bytes: timestamp_nanos (u64, big-endian, nanoseconds since UNIX epoch)
+/// - N bytes: padding (filled with connection_id byte pattern for verification)
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct EchoPayload {
+    /// Unique identifier for the connection that sent this payload.
+    pub connection_id: u64,
```

Comment:
What is a connection ID here? Are we building our own framing protocol for concurrent connections? If we want concurrent connections, we need to open multiple sockets, otherwise we are not actually testing anything useful on the connlib side because it will still all go through the same port number.